Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2749,3 +2749,61 @@
     factions:
     - Passive
 
+- type: entity
+  name: armadillo
+  parent: SimpleMobBase
+  id: MobArmadillo
+  description: some sort of armored rat.
```

Comment:
```suggestion
  description: Some sort of armored rat.
```

Reply:
Done! I made a github boo boo though, sorry for the re-review!